modules/programs/firefox.nix

```
@@ -228,6 +233,14 @@ in {
         '';
       };
 
+      nativeMessagingHosts = mkOption {
+        type = types.listOf types.package;
+        default = [ ];
+        description = mdDoc ''
+          Additional packages containing native messaging hosts that should be made available to Firefox extensions.
+        '';
```

No need to use `mdDoc`:

```suggestion
        description = ''
          Additional packages containing native messaging hosts that should be
          made available to Firefox extensions.
        '';
```